src/components/Router/CrossDeviceMobileRouter.tsx

```diff
@@ -234,10 +234,24 @@ export default class CrossDeviceMobileRouter extends Component<
       return this.setError()
     }
 
+    // NOTE, This is should be removed as soon as motion is supported on Mobile
```

Comment:
Does your feature exists in Orchestration? If yes, how do you handle the case.

Reply:
AFAIK, no, Motion is only being A/B-tested by swapping the "face" step, no exposed through Orchestration.